packages/vite/src/utils.js

```diff
@@ -1 +1,2 @@
-export { compareSignatures } from '@prefresh/utils';
+import { compareSignatures as compare } from '@prefresh/utils';
```

Comment:
This is quick workaround for bundt – didn't realize I was missing `export { ... } from ...` support.

It's effectively the same thing anyway, for the time being.